CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
-# dbt_amazon_ads v0.UPDATE.UPDATE
+# dbt_amazon_ads v0.2.0
+## 🚨 Breaking changes
+- This release is labeled breaking since changes made in the [dbt_amazon_ads_source](https://github.com/fivetran/dbt_amazon_ads_source) are breaking. Note that columns in this package are unchanged.
+- Further details are available in the [dbt_amazon_ads_source CHANGELOG](https://github.com/fivetran/dbt_amazon_ads_source/blob/main/CHANGELOG.md).
+## 🎉 Features
+[PR #6](https://github.com/fivetran/dbt_amazon_ads/pull/6) includes the following updates:
+- Updated documentation to reference changes in the source package.
 
- ## Under the Hood:
+ ## 🚘 Under the Hood
+[PR #6](https://github.com/fivetran/dbt_amazon_ads/pull/6) includes the following updates:
+- Update casts on joins to bridge the gap between ID types changed to STRING and IDs that remained type INTEGER.
```

Comment:
I agree with you that this is a bit nuclear of a solution. It is strange that we now sync the history tables as string datatypes, but we did not change the datatype of the report ids 🤔.

Nevertheless, I would recommend we move aware from doing the individual casting in directly in the joins and instead perform the appropriate casting directly in the staging models to ensure the joins work as expected.

My recommendation would be to cast all the report `id` fields as strings in the source. This will be consistent with a similar change we just applied to the history `id` fields. I also confirmed that it seems our other ad packages leverage a string datatype most consistently. This will ensure consistency across packages while also allowing us to avoid the in join casting.

Reply:
Updated!

Reply:
Correct no longer relevant--updated.